app/views/govuk_publishing_components/components/_lead_paragraph.html.erb

```diff
@@ -1,10 +1,17 @@
 <%
   text ||= ""
   inverse ||= false
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  custom_margin_bottom_class =  shared_helper.get_margin_bottom if [*0..9].include?(local_assigns[:margin_bottom])
+
+  classes = "gem-c-lead-paragraph "
+  classes << "gem-c-lead-paragraph--inverse " if inverse
+  classes << custom_margin_bottom_class if local_assigns[:margin_bottom]
 %>
 
 <% if text.present? %>
-  <p class="gem-c-lead-paragraph <% if inverse %>gem-c-lead-paragraph--inverse<% end %>">
+  <p class="<%= classes %>">
```

Comment:
Might be neater to do this element as a `content_tag` and do the classes as `classes = %w(gem-c-lead-paragraph)` etc. but not essential.